### DISCUSSION TITLE
icicle kit stuck at boot showing "starting kernel" after flash new image in kit

### DISCUSSION THREAD

**Discussion by pratap635** [polarfire-soc/polarfire-soc-documentation#263] [Linux build systems for PolarFire SoC] [answered]
hello,

after flashing new image in icicle kit it's not starting it stuck at boot, showing "starting kernel".

**Comment by hughbreslin** [comment, marked as answer]
Hi @pratap635 have you also updated your Linux image? What release of Linux were you using and what release of the reference design are you using? We advise updating the design and Linux image in sync.
